src/main.rs

```diff
@@ -169,9 +167,8 @@ async fn main() {
     }
 
     // Start the server
+    let addr = config.listen;
```

Comment:
nit: should we just use `config.listen` directly? `addr` is a little ambiguous without context, but not a big deal either way

Reply:
Good idea, thanks. Addressed in 1d453d6b2ac09ed750202e3385ab4ae6863360bd.